plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcConnectorTest.java

```diff
@@ -56,6 +69,42 @@ protected Session withSmallRowGroups(Session session)
                 .build();
     }
 
+    @Override
+    protected boolean isFileSorted(String path, String sortColumnName)
+    {
+        OrcReaderOptions readerOptions = new OrcReaderOptions();
+        try (OrcDataSource dataSource = new FileOrcDataSource(new File(path), readerOptions)) {
+            OrcReader orcReader = OrcReader.createOrcReader(dataSource, readerOptions).orElseThrow();
+            String previousMax = null;
+            OrcColumnId sortColumnId = orcReader.getRootColumn().getNestedColumns().stream()
+                    .filter(column -> column.getColumnName().equals(sortColumnName))
+                    .findFirst()
+                    .orElseThrow()
+                    .getColumnId();
+            List<StripeStatistics> statistics = orcReader.getMetadata().getStripeStatsList().stream()
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .collect(toImmutableList());
+            verify(statistics.size() > 1, "Test must produce at least two row groups");
+
+            for (StripeStatistics stripeStatistics : statistics) {
+                // TODO: This only works if the sort column is a String
+                StringStatistics columnStatistics = stripeStatistics.getColumnStatistics().get(sortColumnId).getStringStatistics();
+
+                if (previousMax != null && previousMax.compareTo(columnStatistics.getMin().toStringUtf8()) > 0) {
```

Comment:
My IDE displays a `Method invocation 'toStringUtf8' may produce 'NullPointerException' ` warning here.
I know it doesn't matter, but I also don't like leaving warnings behind, since it makes me less likely notice important warnings next time.